### ISSUE TITLE
Max zoom on a layer does not show a global view

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory#829]
(based on user feedback documented in UserSnap 190...see #rapidfeedbackteam channel in slack)

As a user, when I am viewing a StoryLayer on MapStory from the detail page, I expect that if I hit the "-" icon to zoom out on the map to its maximum extent, I will be able to see the whole Earth. Currently we have a constraint on the zoom-out button that prevents the user from gaining a global perspective.

For example, go to this layer on mapstory.org: https://mapstory.org/layers/geonode:sea_lamprey. Click the minus sign on top-right of the map canvas to zoom layer out. Further extent possible only shows North America:

![screen shot 2017-07-14 at 2 09 19 pm](https://user-images.githubusercontent.com/2573761/28227107-2d343a10-689e-11e7-86ba-c511daeb9cc0.jpg)

(note, if the layer is changed to "full screen" then the whole Earth is visible. It appears we are just "matting" the map canvas on the detail page rather than proportionally scaling so that a global view is possible on the detail page, as the user expects.)

**Comment by emilyashley** [comment]
Yep yep, I think we're setting the min zoom to 3 and max zoom to 16. If I recall, this was put this way to address users pan and zooming off the world or onto just the poles? Mayhaps we wanna let them pan east-to-west at min zoom of 2 but not north/south?

In addition to that, we're saving the center and zoom level based on a composer view --- the viewer is smaller so from the same center, even less of the basemap will be in view. This is something we've tried to address by communicating the viewer proportions in the new composer mock ups.

Related: we don't honor the available min/max zooms of each basemap we offer (i.e Stamen Terrain goes from 4 - 18 if I recall correctly), so some views won't work depending on what the user selects. Unsure if that is something we wanna address in this ticket by broadening the scope, or something to address later... but it will probs make users feel like something is broken.
